temporary_files_cleaner/temporary_files_cleaner.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo apt-get autoremove -y
+sudo apt-get autoclean
+sudo apt-get clean
+sudo journalctl --vacuum-time=3d
+sudo rm -rf ~/.cache/thumbnails/*
```

Comment:
1. What does this do?
2. Does erasing the files here impact the performance of the software/tool using this location?

Reply:
It is used to clear the cache.
Yes for sure, it reduces the performance of the software/tool using the location. It is the basic concept of cache. But the performance reduction is minute and mostly cannot be noticed by the user.

Reply:
Good then!